### ISSUE TITLE
Avoid Distortion

### ISSUE THREAD

**Issue by dorthwein** [datagovsg/hextile#3]
Is there away to avoid grid distortion - i.e. each polygon is the same?

I've tried setting the correction projection, but then II am un able to add the width in....

**Comment by yongjun21** [comment]
There is no distortion if you don't set a center that is very different from your geojson center

**Comment by dorthwein** [comment]
Let me rephrase it.  As I go north, the tiles become elongated.  So while likely a correct meters projection, the visual appearance is different that tiles closer to the equator, where in our case, I'd prefer to have the meters be different, but visually, the hexagons appear identical.

**Comment by yongjun21** [comment]
In this case, you might want to provide your own [projection](https://github.com/datagovsg/hextile#optionsprojection) functions. Eg.

```javascript
const SCALING_FACTOR = n

const options = {
  projection: {
    forward (lnglat) {
      return lnglat.map(v => v / SCALING_FACTOR)
    },
    inverse (xy) {
      return xy.map(v => v * SCALING_FACTOR)
    }
  }
}
```

Let me know if this helps

**Comment by dorthwein** [comment]
thank you for the example.  I was really struggling on deconstructing the projections correctly.  How would you factor in the width setting into this?
